lib/slimerjs-core.js

```diff
@@ -0,0 +1,58 @@
+/**
+ * Helpers made available when requiring this package.
+ */
+var fs = require('fs')
+var path = require('path')
+
+
+/**
+ * Where the slimerjs binary can be found.
+ */
+try {
+  var location = require('./location')
+  exports.path = path.resolve(__dirname, location.location)
+  exports.platform = location.platform
+  exports.arch = location.arch
+} catch(e) {
+  // Must be running inside install script.
+  exports.path = null
+}
+
+
+/**
+ * The version of firefox installed by this package.
```

Comment:
don't install firefox, This should only install slimerjs, which goes and finds it's own Firefox.

Someone else can (but shouldn't) create a package that install Firefox.